cmd/git/git.go

```diff
@@ -572,6 +573,14 @@ type DSGit struct {
 	Logger logger.Logger
 }
 
+// PublisherPushEvents - this is a fake function to test publisher locally
+// FIXME: don't use when done implementing
+func (j *DSGit) PublisherPushEvents(ev, ori, src, cat, env string, v []interface{}) error {
+	data, _ := jsoniter.Marshal(v)
```

Comment:
let's consider the error here.

Reply:
Ok will add but note that this is only a debug func, this marshaling is only needed for output.

Reply:
Added.